certified-connectors/SignHost/apiProperties.json

```diff
@@ -16,6 +16,9 @@
       }
     },
     "iconBrandColor": "#FFFFFF",
```

Comment:
please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).